src/styles/bootstrap.scss

```diff
@@ -171,15 +172,19 @@
     /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
     @extend .form-control;
 
-    border: .125rem solid $os-default-answer-border-color;
+    border: .125rem solid $os-default-gray;
     border-radius: .3125rem;
-    box-shadow: 0 .125rem .125rem $os-default-box-shadow;
+    box-shadow: 0 .125rem .125rem $os-default-gray;        
   }
 
   input.os-form-control {
     padding: .75rem;
   }
 
+  textarea.os-form-control {
+    padding: 1rem 1.5rem;
+  }
+
```

Comment:
We were previously letting bootstrap determine the padding for the `textarea`, right? If that's the case then this makes sense. I didn't see anywhere else where we were setting the padding for said element.

Reply:
The acceptance criteria says "Change input form padding 24 left/right, 16 top/bottom".  Did I interpret that incorrectly?

Reply:
Yeah I think before It was defaulting to some value provided by bootstrap.